### PR TITLE
Search widgets

### PR DESCRIPTION
**Pull request by carolinux**

### CODE REVIEW

**Comment by rduivenvoorde** [comment]
hi @carolinux looking at the travic cl error we miss a source file?

https://travis-ci.org/qgis/QGIS/builds/66736791

**Comment by carolinux** [comment]
ok, will do! I thought I opened the pull request in my fork, so it's not ready for comprehensive review yet.

File: src/app/qgsattributetabledialog.cpp [hunk 1]
```diff
@@ -706,39 +706,15 @@ void QgsAttributeTableDialog::filterQueryChanged( const QString& query )
...
-    const QgsFields& flds = mLayer->pendingFields();
-    int fldIndex = mLayer->fieldNameIndex( fieldName );
-    if ( fldIndex < 0 )
-      return;
-
-    QVariant::Type fldType = flds[fldIndex].type();
-    bool numeric = ( fldType == QVariant::Int || fldType == QVariant::Double || fldType == QVariant::LongLong );
 
-    QString sensString = "ILIKE";
+    QString sensString = "ILIKE"; // skatoules
```
Comment (jef-n) [inline src/app/qgsattributetabledialog.cpp]: what?

Comment (carolinux) [inline src/app/qgsattributetabledialog.cpp]: it's not ready for review. I submitted it to upstream by mistake.

**Pull request closed without merging**